### DISCUSSION TITLE
413 FILE TOO LARGE

### DISCUSSION THREAD

**Discussion by fred8044** [nextcloud/all-in-one#1738] [Questions] [answered]
### Steps to reproduce
1.Install Nextcloud AIO on Unraid with Docker
2.Install Nextcloud on Ios 16
3.Upload Video

### Expected behavior 
Video upload
### Actual behavior 
Video not upload >> 413 file too large
![IMG_2D5A749887A6-1](https://user-images.githubusercontent.com/122576969/212202618-67333e05-7877-4ae2-a602-a7cf9a79b559.jpeg)

### Host OS 
Unraid 6.11.5

#### Nextcloud AIO version 
25

**Comment by szaimen** [comment]
Hi, are you using a Reverse Proxy or maybe a Cloudflare Tunnel?

**Reply by fred8044** [reply]
yes, cloudflare proxy and nginx proxy manager

**Comment by Zoey2936** [comment, marked as answer]
Cloudflare only allows uploading files smaller than 100MB

**Reply by Zoey2936** [reply]
See [here](https://github.com/nextcloud/all-in-one/discussions/975#discussioncomment-4621980) and the thread after this

**Reply by fred8044** [reply]
Oh, thanks, do u have an alternative or it's necessary use Cloudflare for home use ?

**Reply by Zoey2936** [reply]
> Oh, thanks, do u have an alternative or it's necessary use Cloudflare for home use ?

The only option you have is to remove any kind of Cloudflare proxy

**Reply by fred8044** [reply]
> > Oh, thanks, do u have an alternative or it's necessary use Cloudflare for home use ?
> 
> The only option you have is to remove any kind of Cloudflare proxy

Is it necessary to protect yourself for domestic use?
Do you use a vpn?

**Reply by Zoey2936** [reply]
> > > Oh, thanks, do u have an alternative or it's necessary use Cloudflare for home use ?
> > 
> > 
> > The only option you have is to remove any kind of Cloudflare proxy
> 
> Is it necessary to protect yourself for domestic use? Do you use a vpn?

I think that is a decision that every person needs to decide for themselves.
My AIO instances runs on a hosted server behind my own fork of the nginx-proxy-manager with no Cloudflare.
But if you run AIO at home you have no protection at all, on a hosted server the provides have at least a bit of protection through the provider.
I use a VPN in some scenarios, but in general at home not, what scenario do you mean?

**Reply by fred8044** [reply]
> > > > Oh, thanks, do u have an alternative or it's necessary use Cloudflare for home use ?
> > > 
> > > 
> > > The only option you have is to remove any kind of Cloudflare proxy
> > 
> > 
> > Is it necessary to protect yourself for domestic use? Do you use a vpn?
> 
> I think that is a decision that every person needs to decide for themselves. My AIO instances runs on a hosted server behind my own fork of the nginx-proxy-manager with no Cloudflare. But if you run AIO at home you have no protection at all, on a hosted server the provides have at least a bit of protection through the provider. I use a VPN in some scenarios, but in general at home not, what scenario do you mean?

I speak for example of a ddos ​​attack, at home I have a dream machine pro(unify controller), it may be enough?

**Reply by Zoey2936** [reply]
It depends on the extent of the attack

**Comment by Slopz** [comment]
My uploads keep failing, I'm also behind Cloudflare...are chunked uploads not a thing? I'm confused.

**Reply by szaimen** [reply]
Hi, see the third point of https://github.com/nextcloud/all-in-one#notes-on-cloudflare-proxytunnel

**Reply by Slopz** [reply]
> Hi, see the third point of https://github.com/nextcloud/all-in-one#notes-on-cloudflare-proxytunnel

I understand Cloudflare's limitations...but what I don't get is why I can upload large files when I'm logged into Nextcloud, but can't if I share the folder with someone and they're not logged in. My uploads are chunked, theirs aren't I guess?

**Reply by szaimen** [reply]
> My uploads are chunked, theirs aren't I guess?

Correct
